src/routes/mainRouter.js

```diff
@@ -9,5 +15,7 @@ const router = express.Router();
 router.post("/signup", signupDataValidate, signUp);
 router.route("/login").post(loginDataValidate, logIn).get(authenticateUser);
 router.get("/logout", logOut);
+router.put("/updateuser", authenticateUser, updateUser);
```

Comment:
I would like to recommend to use a route naming convention that was suggest by Elijah in Slack. Let me know if you have questions about it